### ISSUE TITLE
missing ls

### ISSUE THREAD

**Issue by tiago4orion** [c0defellas/enzo#3]

**Comment by lborguetti** [comment]
Guys, what common parameters you use with "ls" program?

**Comment by katcipis** [comment]
-l (to list by line)
-h (human, because well...Im human :-)

**Comment by tiago4orion** [comment]
same here.

but, -h only makes sense with -l and is only used to print human-readable sizes (1k, 1M, 1G, etc) ... then maybe we can default the -l to "human"..
Then, our `ls -l` could output files by line printing human-readable sizes.

The most useful feature of ls must be being sane and safe for scripts... `ls` without arguments must output files/dirs separated by spaces and quote filenames containing spaces.

```sh
λ> touch "filename with spaces"
λ> ls
build-image.sh	dev.json  dev.sh  Dockerfile  'filename with spaces'  Makefile	README.md  tmp
```

**Comment by katcipis** [comment]
makes sense for me, I have no use for sizes in bytes when I'm using ls

**Comment by lborguetti** [comment]
@katcipis don't use size in bytes to do math training? nhaa!

**Comment by katcipis** [comment]
I'm terrible with math :-(

**Comment by tiago4orion** [comment]
**must show hidden files by default**

we can add an option (-n ?) to not show hidden files..

**Comment by katcipis** [comment]
I'm not sure if I like hidden files to not show, would do it later only if necessary

**Comment by tiago4orion** [comment]
Agreed. It's good to always see the hidden garbage..

Then the only parameter required is "-l" ?

@cadicallegari

**Comment by tiago4orion** [comment]
Closed by #19
